mlrun/common/runtimes/constants.py

```diff
@@ -193,8 +195,8 @@ def not_allowed_for_deletion_states():
 
 
 class RunLabels(enum.Enum):
-    owner = "owner"
-    v3io_user = "v3io_user"
+    owner = MlrunInternalLabels.owner
+    v3io_user = MlrunInternalLabels.v3io_user
```

Comment:
Add a comment here, was about to suggest removing this and only using the new constants, but that would break BC.
So let's add a deprecation warning and remove this in 1.9.0